code/todo-app/part1/03-add-dynamodb/chalicelib/db.py

```diff
@@ -116,7 +116,7 @@ def delete_item(self, uid, username=DEFAULT_USERNAME):
     def update_item(self, uid, description=None, state=None,
                     metadata=None, username=DEFAULT_USERNAME):
         # We could also use update_item() with an UpdateExpression.
-        item = self.get_item(uid, username)
+        item = self._table.get_item(uid, username)
```

Comment:
It doesn't look like this change will work.  Calling the underlying `get_item()` method on the ddb client requires a kwarg of `Key`.  We want to use the `self.get_item()` for us because it accepts the params we need and it also returns an the `response['Item']` which is what we want.